data.sql

```diff
@@ -22,3 +22,38 @@ UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+  (1, 1),
+  (3, 1),
+  (3, 2),
+  (4, 2);
+
```

Comment:
![image](https://github.com/edgardsoza/vet-clinic-database/assets/93039359/c0f7208e-155e-4407-8c74-181c14243ec2)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍